package.json

```diff
@@ -149,6 +149,7 @@
     "tween.js": "16.3.1",
     "twemoji": "2.3",
     "uglify-js": "2.7.0",
```

Comment:
I don't think this is required anymore. I removed it, did `update-deps`, tested dev and docker and everything seems to work correctly.

Reply:
woo

Reply:
good point! I'll remove and update deps